app.py

```diff
@@ -1,11 +1,11 @@
 from datetime import datetime
-from io import BytesIO
 import os
 from tempfile import NamedTemporaryFile
 
 from flask import Flask, request, send_file
 import ocrmypdf
-
+from PIL import Image
+import tempfile
```

Comment:
1. I would group the import of standard libs together and keep the 3rd party below
2. `from tempfile import NamedTemporaryFile` already exists